### PR TITLE
Add testcase for backup volume deletion, and function to wait for backup completion

### PR DESCRIPTION
**Pull request by ywei88**
https://github.com/longhorn/longhorn/issues/598

### CODE REVIEW

File: manager/integration/tests/test_basic.py [hunk 1]
```diff
@@ -752,6 +752,47 @@ def restore_inc_test(client, core_api, volume_name, pod):  # NOQA
...
+    volName = generate_volume_name()
+    volume = create_and_check_volume(client, volName)
+
+    volume.attach(hostId=lht_hostId)
+    volume = common.wait_for_volume_healthy(client, volName)
+
+    bv, _, snap1, _ = create_backup(client, volName)
+    _, _, snap2, _ = create_backup(client, volName)
+
+    found = False
```
Comment (yasker) [inline manager/integration/tests/test_basic.py]: Do we have a function to wait for the backup completion?

Comment (ywei88) [reply manager/integration/tests/test_basic.py]: I think no

Comment (yasker) [reply manager/integration/tests/test_basic.py]: Then we can add one.

File: manager/integration/tests/common.py [hunk 2]
```diff
@@ -194,6 +194,8 @@ def create_backup(client, volname, data={}, labels={}):
     # have added extra Labels (for things like BaseImage).
     for key, val in labels.iteritems():
         assert new_b["labels"].get(key) == val
+
+    volume = wait_for_backup_completion(client, volname, snap["name"])
```
Comment (yasker) [inline manager/integration/tests/common.py]: I think we should have code removed because of adding this function?

Comment (ywei88) [reply manager/integration/tests/common.py]: I checked all the places where create_backup is called, and there is no redundant logic except the place we talked before.

File: manager/integration/tests/test_basic.py [hunk 3]
```diff
@@ -784,6 +784,7 @@ def test_deleting_backup_volume(clients):  # NOQA
     bv = client.by_id_backupVolume(volName)
     client.delete(bv)
     common.wait_for_backup_volume_delete(client, volName)
+
```
Comment (yasker) [inline manager/integration/tests/test_basic.py]: extra blank line.

**Pull request merged**